static/js/user_status.js

```diff
@@ -49,6 +49,27 @@ exports.get_status_text = function (user_id) {
     return user_info.get(user_id);
 };
 
+exports.get_status_emoji = function (user_id) {
+    const status_text = user_info.get(user_id);
+    switch (status_text) {
+        case "In a meeting":
+            return "1f4c5"; // calendar
+        case "Commuting":
+            return "1f697"; // automobile
+        case "Out sick":
+            return "1f915"; // head bandage
+        case "Vacationing":
+            return "1f334"; // palm tree
+        case "Working remotely":
+            return "1f4bb"; // laptop
+        default:
+            if (status_text) {
+                return "1f4ac"; // speech balloon
+            }
+            return "";
+    }
+};
```

Comment:
Is the idea here that this model of hardcoding emoji lets us avoid adjusting the database to add a column for status?

Reply:
I can test-deploy this on chat.zulip.org for folks to play with it if so -- but we'll of course want to build a full feature.  Have you QAed it enough that you'd encourage test-deploying?

Reply:
yes, the hard coding was to avoid database changes and I'd encourage test deploying.